netebpfext/net_ebpf_ext_xdp.c

```diff
@@ -3,7 +3,7 @@
 
 /**
  * @file
- * @brief This file implements the XDP program type hook and helper functions on eBPF for Windows.
+ * @brief This file implements the XDP_TEST program type hook and helper functions on eBPF for Windows.
```

Comment:
Since netebpfext should not implement XDP (as xdp-for-windows should be that extension), I think we should remove this file alltogether from netebpfext.  It'd be ok to add it to the sample extension though.  I.e., maybe move it under the ebpf-for-windows\tests\sample\ext\drv directory.

Reply:
As the long term plan is to remove the XDP functionality entirely from this repo (once it has been fully migrated to xdp-for-windows) I am thinking of leaving this functionality within netebpfext for now, with the plan to remove it once xdp-for-windows support is completed. Do you have issues with this approach? (I can file a follow up issue to track this removal).

Reply:
Doing it in multiple steps is fine with me, if you update the PR description so this doesn't "close" 2974.

Reply:
I have added a note in #2974 that it will be done in two parts, with this PR being the first one. I have also updated this description to mention that it addresses part of #2974.